### PR TITLE
feat: add additional pipeline features to deploy feature branches

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: .github/workflows/02-deploy-gh-pages.yml [hunk 1]
```diff
@@ -10,19 +20,67 @@ jobs:
...
+
+      - name: 🖨️ Print gh-pages url
+        env:
+          RELEASE: ${{ inputs.release }}
+          PRE_RELEASE: ${{ inputs.preRelease }}
+          NAME: ${{ steps.extract.outputs.name }}
+        run: |
+          if [[ $RELEASE == "true" ]];
+          then
+            export URL=https://db-ui.github.io/elements
```
Comment (dkolba) [inline .github/workflows/02-deploy-gh-pages.yml:72]: `/elements` here should be `/base`

File: .github/actions/cancel-workflow/action.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    description: "A Github PAT"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        gh api \
+          --method POST \
+          -H "Accept: application/vnd.github+json" \
+          /repos/db-ui/elements/actions/runs/${{ github.run_id }}/cancel
```
Comment (dkolba) [inline .github/actions/cancel-workflow/action.yml:15]: /elements

File: .github/scripts/build-gh-page.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  mkdir temp
+  echo "    Created 'temp' dir"
+else
+  mkdir public
+  echo "    Created 'public' dir"
+fi
+
+
+echo "📥 Get gh-pages tar"
+curl -L https://github.com/db-ui/elements/tarball/gh-pages --output gh-pages
```
Comment (dkolba) [inline .github/scripts/build-gh-page.sh:19]: /elements

**Review by dkolba — CHANGES_REQUESTED** [review]
For minor changes see below - rest LGTM

File: .github/workflows/02-deploy-gh-pages.yml (hunk 1 above)
Comment (nmerget) [reply .github/workflows/02-deploy-gh-pages.yml]: fixed

File: .github/scripts/build-gh-page.sh (hunk 3 above)
Comment (nmerget) [reply .github/scripts/build-gh-page.sh]: fixed

File: .github/actions/cancel-workflow/action.yml (hunk 2 above)
Comment (nmerget) [reply .github/actions/cancel-workflow/action.yml]: fixed

**Pull request merged**